pkg/registry/lock/etcd/etcd_test.go

```diff
@@ -0,0 +1,326 @@
+/*
```

Comment:
Can you please update this file to use "standard" tests instead of instead of reinventing the wheel? You can look at any file at pkg/registry/*/etcd/etcd_test.go.